packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -167,6 +182,37 @@ const useTransactionHistoryWithoutStatuses = (address: string | undefined) => {
   const [depositsLoading, setDepositsLoading] = useState(true)
   const [withdrawalsLoading, setWithdrawalsLoading] = useState(true)
 
+  const cctpTransfersMainnet = useCctpFetching({
+    walletAddress: address,
+    l1ChainId: ChainId.Ethereum,
+    l2ChainId: ChainId.ArbitrumOne,
+    pageNumber: 0,
+    pageSize: 1000,
```

Comment:
We use max page size, which should be enough for 99,99% use cases. Long term we'd have to make changes to CCTP hooks just like in the previous case.